dags/ingest_validated_dataset_to_db.py

```diff
@@ -27,29 +27,40 @@
 )
 
 
-def check_for_validation(**kwargs):
+def _get_first_dagrun():
```

Comment:
Cf commentaire, est-ce qu'on ne peut pas refléter la requête dans la signature de la fonction ? 
Quelque chose comme : 

```suggestion
def _get_first_dagrun_to_insert():
```